### DISCUSSION TITLE
Issue Template

### DISCUSSION THREAD

**Discussion by talmurshidi** [HackYourFutureBelgium/template-markdown#1] [Ideas]
The learners have confusion when creating an issue, there are two options: 
**help-wanted** and **question**, I'm thinking of adding another **issue** template based on issue template in [group-introduction-repo](https://github.com/HackYourFutureBelgium/workflows/blob/master/deliverables/group-introduction-repo.md)

### Update:
1. Added Description Section
2. Update writing the content.
3. moved the issue to the project board and added it to the appropriate column

```
---
name: issue
about: A template issue for features you'd like to add/update/delete.
title: "action: feature: details:"
labels: ""
assignees: ""
---

<!--
  Make your issue easy to find:

  - milestone: the current module
  - labels: anything that will make this easier to filter
-->

### Description

A clear and concise description of what the issue is about.

<!-- You can modify any parts of this template not applicable to your Issue. -->

- Setup: GitHub
  - [ ] created this issue
  - [ ] assigned myself
  - [ ] moved the issue to the project board and added it to the appropriate
        column
- Setup: local
  - [ ] pulled the most recent code from `main`/`master`
  - [ ] created a new `feat-branch-name` branch for my feature 
- Steps to complete the task:
  <!-- You can add the required steps and details to complete the task -->
  - [ ] step 1 details
  - [ ] step 2 details
- Before pushing my branch:
  - [ ] pulled the most recent code from `main`/`master` to my feature branch
  - [ ] ran code quality checks
  - [ ] formatted the code
  - [ ] fixed all linting mistakes
- After pushing my branch:
  - [ ] created a new PR linked to this issue
  - [ ] make sure all of the checks passed
  - [ ] assigned someone to review and merge my PR (if the task is ready)
  - [ ] moved the issue to the appropriate column in the project board to review
        it (if the task is ready)
```

@colevandersWands
@samirm00 
@danielhalasz 
@bermarte

**Comment by sammou00** [comment]
_moved the issue to the project board  and added it to the appropriate column_

**Comment by sammou00** [comment]
- Before pushing my branch:
    - [ ] pulled the most recent code from `main`/`master` to my feature branch
    - [ ] formatted the code
    - [ ] fixed all linting mistakes

maybe add also run code quality check, because we have scripts to validate `HTML` and `CSS`.

**Reply by talmurshidi** [reply]
- Before pushing my branch:
    - [ ] pulled the most recent code from `main`/`master` to my feature branch
    - [ ] ran code quality checks
    - [ ] formatted the code
    - [ ] fixed all linting mistakes

**Reply by talmurshidi** [reply]
Noted, I will add validate `HTML` and `CSS` in template-html-css

**Comment by sammou00** [comment]
Thank you @talmurshidi  It will help `the learners` a lot.
